lrv/app/Models/User.php

```diff
@@ -3,10 +3,10 @@
 namespace App\Models;
 
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Foundation\Auth\User as Authenticatable;
 
-class User extends Model implements Authenticatable
-{
+class User extends Authenticatable
```

Comment:
This change is sane. The only comment I have is maybe change the alias of `Illuminate\Foundation\Auth\User` to `BaseUser` instead of `Authenticatable` as 
`Authenticatable`, `Authorizable` etc. are usually names given to interfaces.

I am happy for this fix to be merged into master. 

Thx @brunotag

Reply:
Yup I'll change that, merge and rerelase, and tag because it's cool.
BTW this rain sucks